app/jobs/inboxes/fetch_imap_emails_job.rb

```diff
@@ -32,16 +32,11 @@ def fetch_mail_for_channel(channel)
                               enable_ssl: channel.imap_enable_ssl
     end
 
-    new_mails = false
-
-    Mail.find(what: :last, count: 10, order: :desc).each do |inbound_mail|
-      next unless inbound_mail.date.utc >= channel.imap_inbox_synced_at
+    Mail.find(what: :last, count: 60, order: :asc).each do |inbound_mail|
```

Comment:
since we are switching to 1-minute intervals for mail fetch. Should we go for a less aggressive number for the count like the original `10` ?
if the jobs end up taking more time iterating over all the mails. it could clog up the workers?

Reply:
@sojan-official what would be the possible number of emails per minute?

Reply:
We can stick to 10 I guess. since we compensate with a quicker polling interval. lets get feedback and then iterate